numpy/core/src/multiarray/datetime.c

```diff
@@ -3910,8 +3913,7 @@ datetime_to_timedelta_resolve_descriptors(
         PyArrayMethodObject *NPY_UNUSED(self),
         PyArray_DTypeMeta *dtypes[2],
         PyArray_Descr *given_descrs[2],
-        PyArray_Descr *loop_descrs[2],
-        npy_intp *NPY_UNUSED(view_offset))
```

Comment:
This looks suspicious.

Reply:
this looks unrelated, no idea what happened here.  will revert.